css/main.css

```diff
@@ -0,0 +1,399 @@
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url("../media/png/gtime.png");
+}
+
+.weather {
+  background-image: url("../media/png/weather.png");
+}
+
+.conference {
+  background-image: url("../media/png/conference.png");
+}
+
+.katchy {
+  background-image: url("../media/png/katchy.png");
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+.contact {
+  width: 100%;
+  height: 95vh;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+  font-family: 'Roboto', sans-serif;
+  font-size: 20px;
+  padding-top: 120px;
+}
+
+.call-to-action {
+  width: 90%;
+}
+
+.contact-form {
+  margin-top: 40px;
+  width: 90%;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  font-size: 20px;
+  margin: 20px 0;
+  width: 100%;
+}
+
+textarea {
+  font-family: sans-serif;
+  background-color: rgba(235, 208, 167, 0.3);
+}
+
+input,
+textarea {
+  padding: 10px;
+  width: 100%;
+  border-bottom: 1px solid grey;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  outline: none;
+}
+
+.btn-primary {
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+  font-size: 15px;
+  padding: 15px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+}
+
+.btn-primary:hover {
+  cursor: pointer;
+}
```

Comment:
- [ ] As the previous reviewer mentioned is important to implement the button interactions (enable, hover, etc.), implementing these interactions will make your UX better :sunglasses:. Please make sure to follow the Figma template.
![image](https://user-images.githubusercontent.com/28109626/147372583-248fba9f-35c3-4925-80c1-0145541ced54.png)